pkgs/development/python-modules/dopy/default.nix

```diff
@@ -0,0 +1,27 @@
+{ pkgs
```

Comment:
use lib instead of pkgs

Reply:
Glad to know this is the preferred attribute to pass. I will use that from now on. Thanks!

Reply:
In this case `pkgs` is needed to use `fetchFromGithub`.